model.py

```diff
@@ -10368,6 +10410,7 @@ class ExternalIntegration(Base, HasFullTableCache):
         AXIS_360 : DataSource.AXIS_360,
         RB_DIGITAL : DataSource.RB_DIGITAL,
         ENKI : DataSource.ENKI,
+        FEEDBOOKS : DataSource.FEEDBOOKS,
```

Comment:
Without this, calling `Collection.data_source` on a Feedbooks collection will return None.